.travis.yml

```diff
@@ -5,7 +5,6 @@ addons:
   apt:
     sources:
       - ubuntu-toolchain-r-test
-#      - llvm-toolchain-precise-3.7
```

Comment:
We could probably replace this with `llvm-toolchain-trusty-6.0` and go up to Clang 6.0 instead. I'm not blocking the PR on this though.

Reply:
Moved to 7.